Author: Andreas Hoenen <andreas@hoenen-terstappen.de>
Description: hotfix for BTS report #840189:
Adapt to signature change of \@xmultirow macro (between texlive-latex-extra
versions 2016.20160819-1 and 2016.20161008-1).
--- a/latex/style/dbk_table.sty
+++ b/latex/style/dbk_table.sty
@@ -24,12 +24,13 @@
 % Make \@xmultirow long
 \expandafter\long\expandafter\def%
 \expandafter\@xmultirow%
-\expandafter#\expandafter1%
-\expandafter[\expandafter#\expandafter2\expandafter]%
-\expandafter#\expandafter3%
-\expandafter[\expandafter#\expandafter4\expandafter]%
-\expandafter#\expandafter5%
-\expandafter{\@xmultirow{#1}[#2]{#3}[#4]{#5}}
+\expandafter[\expandafter#\expandafter1\expandafter]%
+\expandafter#\expandafter2%
+\expandafter[\expandafter#\expandafter3\expandafter]%
+\expandafter#\expandafter4%
+\expandafter[\expandafter#\expandafter5\expandafter]%
+\expandafter#\expandafter6%
+\expandafter{\@xmultirow[#1]{#2}[#3]{#4}[#5]{#6}}
 
 % For the newtbl code
 \newdimen\newtblstarfactor%
